en/docs/observability/root-cause-analysis.md

```diff
@@ -0,0 +1,99 @@
+
+### Analysing the throughput graph
+The throughput graph will show the throughput of requests per hour for a selected timestamp. 
+
+With the sample data, you will observe spikes in the throughput graph. Let's see how you can find
+the root cause by analyzing this data.
+
+Follow the steps below. 
+
+1. Hover over the throughput graph at 01/04/2021 13:41:00. You can observe that 46.06 errors had occurred at the time. 
+    
+    !!!tip
+        You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
+        
+2. Click on the graph on 01/04/2021 at 13:41:00.
+3. Observe the logs in the logs panel.
+4. Note multiple logs are mentioning a connection error with the hr-service.
+ 
+![Analyzing the throughput graph](/assets/img/observability/throughput-graph-analysis.png)
+ 
+### Analyzing the latency graph
+
+The latency graph shows the latencies of each request.
+
+With the sample data, you will observe spikes in the latency graph. By analyzing the latency graph, you can identify which API invocation caused the error.  Let's see how to find the
+root cause by analyzing this data.
+![Analyzing the latency graph](/assets/img/observability/latency-graph-analysis.png)
+
+Follow the steps below.
+
+1. Hover over the latency graph and click at 01/04/2021 13:41:00.
+2. Observe the latencies of each request listed in the low-code diagram.
+3. Click on one request shown on the low code diagram (latencies). You can observe the status code for that request and thereby identify the exact API invocation that caused any error(s).
+
+### Diagnostics View
+The Diagnostics view provides the capability to view errors, throughput, latency, CPU, memory, and logs simultaneously for a particular event. You can detect and analyze errors and anomalies in detail via the diagnostics view.
+
+![Diagnostic View](/assets/img/observability/diagnostics-view.png)
+
+We refer to a horizontal section of the graph for a particular period as the **bin**. A bin consists of the following.
+
+- **Start time and end time**- Denoted by the dotted lines separating each bin.
+- **Logs**- Any logs that appeared during the bin's time range are listed on the logs column, along with the timestamp the logs started to appear and the log count for each log. The logs are listed according to precedence. The error logs will be listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted based on the log count.
+- **Error**- Number of HTTP errors occurred at the selected time.
+- **TP**- Throughput of the requests at the selected time (req/s).  
+- **Latency**- Latency of the request at the selected time (ms).
+- **CPU**- CPU usage at the selected time (milicores).
+- **Memory**- Memory usage at the selected time (MiB)
+
+Let's see how to find the root cause by analyzing this data.
+
+Follow the steps below.
+
+1. Click on the **Diagnostics View** tab on the left panel. 
+
+    !!!info
```

Comment:
Shall we use `>**Info:**` to mark the info things?

Reply:
Using this gives no color coding to the admonitions, right? For e.g. Info, warning, tip, error will all be displayed the same, isn't it? The inbuilt admonitions for the material theme, has this color theme.

Reply:
![image](https://user-images.githubusercontent.com/16300038/115666256-f905e380-a361-11eb-8c44-a7ccec313870.png)

It will like above. Not sure whether it is ok for this context.

Reply:
yes... and it will look the same for all other admonitions. There will be no difference. Using the inbuilt  admonitions will help us use the color coding and thereby increasing readability and grasping the attention of the user.